### PR TITLE
GEOPY-1252

### PR DESCRIPTION
**Pull request by domfournier**

### CODE REVIEW

File: simpeg_drivers/utils/testing.py [hunk 1]
```diff
@@ -81,7 +81,9 @@ def check_target(output: dict, target: dict, tolerance=0.1):
     :param tolerance: Tolerance between output and target measured as: |a-b|/b
     """
     print(
-        f"'data_norm': {np.linalg.norm(output['data'])}, 'phi_d': {output['phi_d'][1]}, 'phi_m': {output['phi_m'][1]}"
+        f"'data_norm': {np.linalg.norm(output['data'])}, "
```
Comment (benk-mira) [inline simpeg_drivers/utils/testing.py:84]: Let's put the target info in here too

**Review by benk-mira — CHANGES_REQUESTED** [review]
update print statement to include target values

File: simpeg_drivers/utils/testing.py (hunk 1 above)
Comment (domfournier) [reply simpeg_drivers/utils/testing.py:84]: Done

**Review by benk-mira — APPROVED** [review]
thanks!

**Pull request merged**